assets/base.css

```diff
@@ -474,6 +474,11 @@ blockquote {
   letter-spacing: 0.04rem;
 }
 
+h2.caption-large,
+h3.caption-large {
+  font-size: calc(var(--font-heading-scale) * 1.3rem);
+}
+
 .color-foreground {
   color: rgb(var(--color-foreground));
```

Comment:
The price `p` in the gift card doesnt seem to be updating either: https://screenshot.click/28-22-zd38z-zdtqd.png